arches/app/templates/views/components/search/advanced-search.htm

```diff
@@ -18,9 +18,9 @@ <h4 class="search-facet-item-heading">{% trans "Search Facets" %}</h4>
             <!-- ko foreach: searchableGraphs -->
                 <!-- ko if: cards().length > 0  -->
                 <div class="search-facet-item disabled">
-                    <h4 class="search-facet-item-heading">
+                    <h4 class="search-facet-item-heading" data-bind="click: function () { collapsed(!collapsed()) }">
```

Comment:
This seems like it should be a level higher; at the current level it doesn't include the padding for the search facet, so the entire facet isn't clickable.